packages/rx_bloc_cli/lib/src/rx_bloc_cli_constants.dart

```diff
@@ -14,7 +14,8 @@ const kAndroidMinSDKVersion = 22;
 const kGradlePluginVersion = '8.3.2';
 
 /// Generated project's Android Min SDK version
-const kGradleWrapperUrl = 'https\\://services.gradle.org/distributions/gradle-8.4-all.zip';
+const kGradleWrapperUrl =
```

Comment:
Do we need this change ?

Reply:
This was just from auto-formatting, so... no